test/Microsoft.TestPlatform.AcceptanceTests/TestPlatformNugetPackageTests.cs

```diff
@@ -18,11 +18,11 @@ public class TestPlatformNugetPackageTests : CodeCoverageAcceptanceTestBase
         private string resultsDirectory;
 
         [ClassInitialize]
-        public static void ClassInit(TestContext testContext)
+        public static void ClassInit(TestContext _)
         {
             var packageLocation = Path.Combine(IntegrationTestEnvironment.TestPlatformRootDirectory, "artifacts", IntegrationTestEnvironment.BuildConfiguration, "packages");
             var nugetPackage = Directory.EnumerateFiles(packageLocation, "Microsoft.TestPlatform.*.nupkg").OrderBy(a => a).FirstOrDefault();
-            nugetPackageFolder = Path.Combine(packageLocation, Path.GetFileNameWithoutExtension(nugetPackage));
+            nugetPackageFolder = Path.Combine(GetResultsDirectory(), Path.GetFileNameWithoutExtension(nugetPackage));
```

Comment:
In most places you are using workspace.Path instead of this, is this correct?

Reply:
Correct. I forgot to add a comment suggesting if I could rename `GetResultsDirectory` into something like `CreateUniqueDirectory` which makes more sense for what it does.

Reply:
Yes that sounds like a better name.